### PR TITLE
Improve cli for mephisto wut

### PR DESCRIPTION
**Pull request by Etesam913**
## Summary

Improved the cli of the `mephisto wut` command and general cli commands like `mephisto`. 

It should be easier to see options for commands and errors should be more clear.

Tables should be much easier to read.

This is done with the [rich-click](https://github.com/ewels/rich-click) and `rich` packages.

The `rich-click` package gives rich formatting to cli commands. 

```python
import rich_click as click
```
this replaces the default click object to get basic rich formatting. 

The `rich` package is still needed for printing things to the console, such as tables.

## Video
https://user-images.githubusercontent.com/55665282/178544113-1397707d-6b4c-45bd-a598-3d0cbd6cb1b7.mov

### CODE REVIEW

**Comment by JackUrb** [comment]
Ok I really need to fix our deps to get `rich` required correctly - this is awesome!

**Comment by pringshia** [comment]
🤤

File: mephisto/client/cli.py [hunk 1]
```diff
@@ -238,25 +295,38 @@ def get_help_arguments(args):
...
+                    f"The requester is an account for a crowd provider. \n"
+                    "Use `mephisto requesters` to see registered requesters, and `mephisto register <requester type>` to register.\n"
+                )
+                valid_requester_text = """**Valid requesters are:**"""
+                print_out_valid_options(
+                    valid_requester_text, get_valid_provider_types()
                 )
                 return
             elif abstraction == "provider":
+                # TODO: Use markdown list here
```
Comment (JackUrb) [inline mephisto/client/cli.py:324]: Do you have a plan for executing this TODO, or abandoning?

Comment (Etesam913) [reply mephisto/client/cli.py:324]: I already executed it, the print_out_valid_options method uses the markdown list.

I forgot to remove the TODO though.

**Pull request merged**